From 70c640ce992234aacba5a717f3fb47319f451431 Mon Sep 17 00:00:00 2001
From: Phil Elwell <phil@raspberrypi.com>
Date: Thu, 2 May 2024 17:40:25 +0100
Subject: [PATCH 1213/1215] dts: bcm2712-rpi: The SoC gpiochips start at 10

Make the BCM2712's onboard GPIOs start at gpiochip10, marking them out
as system resources and preventing accidental use by existing Pi 5
code.

Signed-off-by: Phil Elwell <phil@raspberrypi.com>
---
 arch/arm64/boot/dts/broadcom/bcm2712-rpi.dtsi | 1 +
 1 file changed, 1 insertion(+)

--- a/arch/arm64/boot/dts/broadcom/bcm2712-rpi.dtsi
+++ b/arch/arm64/boot/dts/broadcom/bcm2712-rpi.dtsi
@@ -113,6 +113,7 @@
 		gpio3 = &pinctrl;
 		gpio4 = &pinctrl_aon;
 		gpiochip0 = &gpio;
+		gpiochip10 = &gio;
 		i2c = &i2c_arm;
 		i2c0 = &i2c0;
 		i2c1 = &i2c1;
